### PR TITLE
ci: add python-sphinx to lyft/envoy-build.

### PR DESCRIPTION
**Pull request by htuch**
Needed by data-plane-api repo (and main envoy eventually) to be able to build both Bazel and sphinx
in a single image.

Signed-off-by: Harvey Tuch <htuch@google.com>

### CODE REVIEW

File: ci/build_container/build_container_ubuntu.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -e
 # Setup basic requirements and install them.
 apt-get update
 apt-get install -y wget software-properties-common make cmake git python python-pip \
-  bc libtool automake zip time golang g++ gdb strace
+  bc libtool automake zip time golang g++ gdb strace python-sphinx
```
Comment (mattklein123) [inline ci/build_container/build_container_ubuntu.sh]: Should we use the same venv that we currently use in the main docs build?

Comment (htuch) [reply ci/build_container/build_container_ubuntu.sh]: My plan was to do this by copying `docs/requirements.txt` to the same place in `data-plane-api`. Or, do you want to set this up in the base image?

Comment (mattklein123) [reply ci/build_container/build_container_ubuntu.sh]: The venv includes sphinx. Do we need this change then?

**Comment by htuch** [comment]
No we do not, thanks for pointing this out.

**Pull request closed without merging**